src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -102,7 +102,7 @@ def skip_table(self, schema: str, table: str):
                 f"ALTER TABLE `{schema}`.`{table}` SET TBLPROPERTIES('{self.UCX_SKIP_PROPERTY}' = true)"
             )
         except NotFound as nf:
-            if "[TABLE_OR_VIEW_NOT_FOUND]" in str(nf):
+            if "[TABLE_OR_VIEW_NOT_FOUND]" in str(nf) or "[DELTA_TABLE_NOT_FOUND]" in str(nf):
                 logger.error(f"Failed to apply skip marker for Table {schema}.{table}. Table not found.")
             else:
                 logger.error(nf)
```

Comment:
This error log needs context (the table name)

Reply:
we print schema and table name. What is missing?

Reply:
added